htdocs/include/checklogin.php

```diff
@@ -48,7 +48,7 @@
 
 icms_loadLanguageFile('core', 'user');
 $uname = !isset($_POST['uname']) ? '' : trim($_POST['uname']);
-$pass = !isset($_POST['pass']) ? '' : trim($_POST['pass']);
+$pass = !isset($_POST['pass']) ? '' : substr(trim($_POST['pass']), 0, 99);
```

Comment:
what if somebody already has pass with length > 99 ? :/

Reply:
if someone has a pass with length >99, I'll need to add a text that says you need to reset your password. Due to the hashing, we cannot know the length of the original password unfortunately